### PR TITLE
Changes for mypy such as ignoring libraries

### PR DESCRIPTION
**Pull request by alfredeen**
## Status

- [x] Ready
- [ ] Draft
- [ ] Hold

## Description

Moved explicit infile mypy ignores of library imports to configurations in the toml file. Also changed some relative imports to package imports to solve mypy rule E0402.

## Types of changes

What types of changes does your code introduce to STACKn?

- [ ] Hotfix (fixing a critical bug in production)
- [ ] Bugfix
- [x] New feature
- [ ] Documentation update

## Checklist

- [x] This pull request is against **develop** branch (not applicable for hotfixes)
- [ ] I have included a link to the issue on GitHub or JIRA (if any)
- [ ] I have included migration files (if there are changes to the model classes)
- [x] I have read the [CONTRIBUTING](https://github.com/scaleoutsystems/stackn/blob/master/CONTRIBUTING.md) doc
- [x] I have included, reviewed and executed tests (unit and end2end) to complement my changes
- [ ] I have updated the related documentation (if necessary) 
- [ ] I have updated the release notes (docs/releasenotes.md)
- [x] I have added a reviewer for this pull request
- [x] I have added myself as an author for this pull request

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -25,3 +25,27 @@ python_version = "3.8"
...
+	"dash.*",
+	"dash_core_components.*",
+	"dash_html_components.*",
+	"django_plotly_dash.*",
+	"dash_bootstrap_components.*",
+	"markdown.*",
+  "requests.*",
+  "s3fs.*",
+	"setuptools.*",
+  "yaml.*",
```
Comment (churnikov) [inline pyproject.toml:49]: Fix indent, please :)

**Review by churnikov — APPROVED** [review]

File: pyproject.toml (hunk 1 above)
Comment (alfredeen) [reply pyproject.toml]: Fixed.

File: apps/helpers/helpers.py [hunk 2]
```diff
@@ -5,9 +5,9 @@
 from django.conf import settings
 from django.template import engines
 
-from ..models import AppInstance, AppStatus  # type: ignore
-from ..serialize import serialize_app  # type: ignore
-from ..tasks import deploy_resource  # type: ignore
+from apps.models import AppInstance, AppStatus
+from apps.serialize import serialize_app
+from apps.tasks import deploy_resource
```
Comment (sandstromviktor) [inline apps/helpers/helpers.py:10]: Great! This has bothered me

**Review by sandstromviktor — APPROVED** [review]
Good job!

File: apps/helpers/helpers.py (hunk 2 above)
Comment (churnikov) [reply apps/helpers/helpers.py:10]: Are you against relative imports? :)

Comment (sandstromviktor) [reply apps/helpers/helpers.py:10]: Hehe well i think so. I like more explicit stuff. But I can change my mind.

Comment (alfredeen) [reply apps/helpers/helpers.py:10]: This also fixes mypy error: Relative import climbs too many namespaces.

**Pull request merged**